Composer/packages/server/src/models/bot/botProject.ts

```diff
@@ -216,7 +217,7 @@ export class BotProject implements IBotProject {
     if (userSDKSchemaFile !== undefined) {
       debug('Customized SDK schema found');
       try {
-        sdkSchema = JSON.parse(userSDKSchemaFile.content);
+        sdkSchema = merge(sdkSchema, JSON.parse(userSDKSchemaFile.content));
```

Comment:
Is there any reason to merge instead of reassign?

Reply:
Yes; this allows for cases like UK or Australian English where strings that don't need to be localized any differently from default can just be left out.

Reply:
I don't think this will work with the PVA integration (as well as any other custom schema) because our base schema is generated from the sdk components, but it is possible for users to modify their schema as they desire. For instance, if a component was removed from the schema, this would merge it back in.

Reply:
Changed this back. It took some time to come to agreement with this, but you're right. If someone has their own custom schema, we don't want to merge on top of that; we can assume that a custom user schema will already be in the language the user needs it to be in.